packages/components/src/internal/components/user/SiteUsersGridPanel.tsx

```diff
@@ -113,97 +128,114 @@ export class SiteUsersGridPanel extends React.PureComponent<Props, State> {
         }
     }
 
+    initQueryModel(usersView: string) {
+        const { actions, user } = this.props;
+        const baseFilters = usersView === 'all' ? [] : [Filter.create('active', usersView === 'active')];
```

Comment:
Does this need to handle the 'inactive' users view?  Is it worthwhile to make constants or an enum for these view values?

Reply:
It does. The `usersView === 'active'` value will results in a "filter for active = false" in the 'inactive' users view case.

Reply:
Ah of course.  Thanks.